views/personal/residence.pug

```diff
@@ -6,24 +6,15 @@ block variables
 
 block content
 
-  h1 #{title}
+  h1 #{__(title)}
 
   div
-    p In order to be eligible for this service, your <strong>primary residence</strong> on December 31, 2018, must have been in Ontario.
+    p #{__('In order to be eligible for this service, your primary residence on December 31, 2018, must have been in Ontario.')}
 
 
   form.cra-form(method='post')
     +yesNoRadios('residence', residence, 'Ontario resident on December 31, 2018?', errors)
```

Comment:
Is the translation module going to be able to figure out these strings that are passed as arguments or do we need to add those to the locale file manually?

Reply:
Yup, it's in there: https://github.com/cds-snc/cra-claim-tax-benefits/pull/49/files#diff-1d62a5823ca4461fb794fa0aa5de49eeR86

We do the same thing with our error messages: https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/views/_includes/errorList.pug#L6